scripts/alert_emails/template.html

```diff
@@ -144,7 +156,7 @@
                             style="line-height:inherit;border-width:0!important;border-style:none!important;border-color:white!important">
                             <div
                                 style="line-height:inherit;Margin:0 auto;max-width:600px!important;min-width:320px;width:600px!important;word-wrap:break-word;word-break:break-word;border-width:0!important;border-style:none!important;border-color:white!important">
-                                <div
+                                <!-- <div
```

Comment:
I'm surprised we're removing this link.  We're intentionally not making it easy to see the data / metrics?

Reply:
I'm triple-checking with Igor, but he did confirm the current structure and copy (and proposed the change from _community level → community risk level_). I'm going to merge to `develop` and submit a separated PR if we finally decide to have this section back.